### ISSUE TITLE
Home not sent with git_add_and_commit shortcut

### ISSUE THREAD

**Issue by michaelmior** [scmbreeze/scm_breeze#156]
A couple examples of what happens. I enter the number of a file and press the defined shortcut (^X c). `git_add_and_commit` is run, but without sending home first. It seems to send a space instead. This is with zsh 5.0.5.

Examples:

```
$ 1 2  git_add_and_commit 
__original_cd:cd:18: too many arguments
$ 1  git_add_and_commit 
__original_cd:cd:18: string not in pwd: -
```

**Comment by robwise** [comment]
@michaelmior I'm getting this also, did you find a solution?

**Comment by michaelmior** [comment]
No, unfortunately I didn't. I've just restored to separately calling `git add`.

**Comment by michaelmior** [comment]
This works if I replace `033[1~` with `^[OH` in `lib/git/keybindings.sh`. I'm not sure why the original binding doesn't work though.

**Comment by robwise** [comment]
@michaelmior Thanks! This worked for me! Cheers
